app/components/newhouseCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+interface HouseHoldCardProps {
+  id: string;
+  name: string;
+  desc?: string;
+  icon?: string; //URL of image, if none provided, default icon is used
+}
+
+function HouseholdCard(props: HouseHoldCardProps) {
+  const icon = props.icon || "https://placeimg.com/148/148/arch";
+
```

Comment:
Perhaps put the "./plus.png" URI here instead of the placeimg URL link, otherwise this icon constant is never used in this component.